src/_includes/section-menu/sdks.html

```diff
@@ -0,0 +1,19 @@
+<li><span class="section-nav-list-main-title">SDKS</span></li>
+<li><a href="{{ site.baseurl }}/documentation/sdks/java/">Beam SDK feature comparison</a></li>
```

Comment:
we should remove the SDK feature comparison link entirely for now, until it exists